src/js/tracks/text-track-list-converter.js

```diff
@@ -28,8 +28,11 @@ let trackToJson_ = function(track) {
         id: cue.id
       };
     }),
-    src: track.src
   };
+  if (track.src) {
```

Comment:
Instead of specifically handling `src` here, would it make sense to generally exclude any falsy property?

```js
Object.keys(ret).forEach(k => {
  if (!ret[k]) {
    delete ret[k];
  }
});
```